src/cli/cli.cpp

```diff
@@ -4023,12 +4060,71 @@ template <> otError Interpreter::Process<Cmd("locate")>(Arg aArgs[])
     otError      error = OT_ERROR_INVALID_ARGS;
     otIp6Address anycastAddress;
 
+    /**
+     * @cli locate
```

Comment:
This one doesn't come through the import because OPENTHREAD_CONFIG_TMF_ANYCAST_LOCATOR_ENABLE isn't set in our Doxygen config